app/cmd/rest.go

```diff
@@ -1,55 +1,39 @@
 package cmd
 
 import (
-	"encoding/json"
 	"fmt"
 	"log"
 	"net/http"
-)
-
-func StartRest() {
-	controller()
-}
 
-func controller() {
-	http.HandleFunc("/health", health)
-	http.HandleFunc("/hello-world", helloWorld)
+	"github.com/nadhiljanitra/nadhil-golang-training-beginner/app/config"
+	"github.com/nadhiljanitra/nadhil-golang-training-beginner/healthcheck"
+	code "github.com/nadhiljanitra/nadhil-golang-training-beginner/paymentcode"
```

Comment:
not really need to use import alias unless conflict

Reply:
i have remove the alias, but when i want to call the package for example `paymentcode.RegisterCtrl()` it always return `paymentcode` not defined sir. any suggestion? is it because in that file, my package name is `code`?